sys/net/crosslayer/ng_netreg/ng_netreg.c

```diff
@@ -101,4 +102,30 @@ ng_netreg_entry_t *ng_netreg_getnext(ng_netreg_entry_t *entry)
     return entry;
 }
 
+ng_pktsnip_t *ng_netreg_hdr_build(ng_nettype_t type, ng_pktsnip_t *payload,
+                                  uint8_t *src, uint8_t src_len,
+                                  uint8_t *dest, uint8_t dest_len)
+{
+    switch (type) {
+#ifdef MODULE_NG_IPV6
+
+        case NG_NETTYPE_IPV6:
+            return ng_ipv6_hdr_build(payload, src, src_len, dest, dest_len);
```

Comment:
Since it is guarded by the `#ifdef MODULE_NG_IPV6` guard this line will only be compiled if the new IPv6 module is too.